client/src/pages/Login.jsx

```diff
@@ -123,13 +124,15 @@ function useLogin() {
   const history = useHistory();
 
   const login = async (email, password) => {
-    console.log(email, password);
-    const res = await fetch(
-      `/auth/login?email=${email}&password=${password}`
-    ).then(res => res.json());
-    localStorage.setItem("user", res.user);
-    localStorage.setItem("token", res.token);
-    history.push("/dashboard");
+    try {
+      const body =JSON.stringify({email,password});
+      const config = {headers: {'Content-Type': 'application/json'}};
+      await axios.post(`/users/login`,body,config);
+      history.push("/dashboard");
```

Comment:
We should check the response to see if it was successful before redirecting to the dashboard.